app/Main.hs

```diff
@@ -155,7 +155,7 @@ argParser = Opt.subparser $ mconcat $
         putStrLn $ "Originating " <> name <> "..."
         env <- mkMorleyClientEnv cconf
         (_, addr) <- runMorleyClientM env $ do
-          lOriginateContract False (mkAliasHint name) (AddressResolved sAdmin) zeroMutez
+          lOriginateContract True (mkAliasHint name) (AddressResolved sAdmin) zeroMutez
```

Comment:
I see that you've change this, was there any particular reason?

I'd say that the non-replacing behavior would be better in case of a mistake.

We can also make this too a command line option BTW :shrug:

Reply:
This apparently got messed up during some rebase and I didn't notice.

So the behaviour with `False` there is really not what we want here: it originates the contract, and _then_ fails, complaining that it can't find the name of the contract it just originated. I'm questioning whether there's a bug in morley-client, but as it is now, replacement is the only reasonable option. The docs also say that the alias will be overwritten, btw.

Reply:
> it originates the contract, and then fails, complaining that it can't find the name of the contract it just originated

:thinking: I don't think that was the intended behavior, but I may be wrong, can you check this and open an issue in `morley`, please?

> The docs also say that the alias will be overwritten, btw.

Yeah, I looked for that but missed it before. Let's keep this behavior then :+1:

Reply:
Looks like a bug to me. I've opened [morley#763](https://gitlab.com/morley-framework/morley/-/issues/763).